pkg/sql/distsql/server.go

```diff
@@ -528,7 +528,10 @@ func (ds *ServerImpl) SetupLocalSyncFlow(
 func (ds *ServerImpl) SetupFlow(
 	ctx context.Context, req *execinfrapb.SetupFlowRequest,
 ) (*execinfrapb.SimpleResponse, error) {
-	log.VEventf(ctx, 1, "received SetupFlow request from n%v for flow %v", req.Flow.Gateway, req.Flow.FlowID)
+	log.VEventf(
+		ctx, 1, "received SetupFlow request from n%d for flow %s",
+		redact.SafeInt(req.Flow.Gateway), redact.SafeString(req.Flow.FlowID.String()),
```

Comment:
These types should be handled already. Don't call String on integer-like values.